tests/nightly/test_np_large_array.py

```diff
@@ -90,28 +90,28 @@ def test_softmax():
 
 @use_np
 def test_ones():
-    A = np.ones((INT_OVERFLOW, 2))
-    assert A.shape == (INT_OVERFLOW, 2)
-    assert A[0][0] == 1
+    data = np.ones((INT_OVERFLOW, 2))
```

Comment:
data -> input ?

Reply:
I think for generative ops maybe we can use `data`? For other ops I will definitely use `input` and `output`

Reply:
`input` is a built-in function of python. I think `data` is better.